documentation/STRUCTURE.md

```diff
@@ -0,0 +1,161 @@
+
+Example:
+
+- [pandas](https://pandas.pydata.org/getting_started.html)
+- [scikit-learn](https://scikit-learn.org/dev/getting_started.html)
+
+### API reference
+
+This is simply the generated documentation from our code. Currently we host this in readthedocs, but we want to include it on our webpage. A good example for this is [scikit-learn](https://scikit-learn.org/dev/modules/classes.html)
+
+### User guide
+
+The user guide explains all parts of Fairlearn by providing context that wouldn't fit into the code documentation such as mathematical derivations, but without using application-specific context (as we'd find it in the "case studies"). The guides are grouped by topic, e.g.
+
+1. Assessment
+    1. Metrics
+        1. ...
+    1. Visualizations
+        1. ...
+1. Mitigation
+    1. Reductions methods
+        1. Exponentiated Gradient
+        1. Grid Search
+    1. Postprocessing
+        1. Threshold Optimizer
+
+Examples:
+
+- [scikit-learn](https://scikit-learn.org/dev/user_guide.html)
+- [pandas](https://pandas.pydata.org/docs/user_guide/index.html)
+
+Of our current notebooks the following may be most suitable as "user guides":
+
+- [Group Metrics](https://github.com/fairlearn/fairlearn/blob/master/notebooks/Group%20Metrics.ipynb) - a great example for something that should be a user guide
+- [Grid Search for Binary Classification](https://github.com/fairlearn/fairlearn/blob/master/notebooks/Grid%20Search%20for%20Binary%20Classification.ipynb) - the purpose is mostly to show Grid Search's functionality; perhaps it may need to be trimmed down to the essentials about Grid Search
+- [Grid Search with Census Data](https://github.com/fairlearn/fairlearn/blob/master/notebooks/Grid%20Search%20with%20Census%20Data.ipynb) - similar to the previous notebook it covers Grid Search; we could leverage some of this for a user guide for Grid Search, or alternatively for the dashboard visualizations
+
+### Case studies
+
+The purpose of fairness case studies is to walk through an application of Fairlearn in detail. Any application of a fairness toolkit needs to be done with great care while taking into account an entire range of concerns due to the sociotechnical nature of fairness. The showcased case studies will provide the space to cover scenarios in depth. The focus is not only on showing example usage of the Fairlearn toolkit, but on how to approach fairness in ML in general. We may want to add scenario even if it contains only few of Fairlearn's capabilities, but it otherwise demonstrates a great example of how to build AI responsibly.
+
+All the case studies should be downloadable as Jupyter notebooks and/or Python source code, and be launchable in [Binder](https://mybinder.org/) and perhaps other platforms.
```

Comment:
I'd say or a similar platform (I think you wanted to have it on azure notebooks or something)

Reply:
So long as the 'or' isn't exclusive :-) I think we would want to provide multiple options for users, if we can.

Reply:
sure, the more the better